pom.xml

```diff
@@ -15,28 +15,40 @@
   <properties>
     <!--Project properties-->
     <java.version>17</java.version>
-    <spring.version>5.3.23</spring.version>
+    <spring.version>6.0.2</spring.version>
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <generated_sources_dir>${project.build.directory}/generated-sources</generated_sources_dir>
     <jsonschema2pojo_output_dir>${generated_sources_dir}/jsonschema2pojo</jsonschema2pojo_output_dir>
     <postgres.image>postgres:12-alpine</postgres.image>
 
     <!--Dependency Management Properties-->
-    <vertx.version>4.3.4</vertx.version>
+    <vertx.version>4.4.5</vertx.version>
     <jackson-bom.version>2.13.4</jackson-bom.version>
-    <log4j.version>2.19.0</log4j.version>
+    <log4j.version>2.20.0</log4j.version>
 
     <!--Dependency properties-->
-    <aspectj.version>1.9.19</aspectj.version>
+    <aspectj.version>1.9.20.1</aspectj.version>
     <aspectj-maven-plugin.version>1.13.1</aspectj-maven-plugin.version>
-    <rest-assured.version>5.2.0</rest-assured.version>
-    <raml-module-builder.version>35.0.1</raml-module-builder.version>
+    <rest-assured.version>5.3.2</rest-assured.version>
+    <raml-module-builder.version>35.1.0</raml-module-builder.version>
+    <folio-di-support.version>2.0.1</folio-di-support.version>
+
+    <!--Maven plugin properties-->
+    <exec-maven-plugin.version>3.1.0</exec-maven-plugin.version>
+    <maven-resources-plugin.version>3.3.1</maven-resources-plugin.version>
```

Comment:
do we really need to put each plugin version into a variable?

Reply:
From my perspective, it is good practice. I am trying to create a pattern among our modules that will help to update maven dependencies or other dependencies easily with one copy past in future.